tapir/statistics/management/commands/process_credit_account.py

```diff
@@ -0,0 +1,75 @@
+import csv
+import datetime
+import fnmatch
+import io
+from typing import Dict
+
+import environ
+import paramiko
+from django.core.management.base import BaseCommand
+from django.db import transaction
+from django.utils import timezone
+from fabric import Connection
+from paramiko.sftp_file import SFTPFile
+
+from tapir.accounts.models import TapirUser
+from tapir.statistics.models import ProcessedCreditFiles, CreditAccount
+from tapir.utils.shortcuts import get_timezone_aware_datetime
+from tapir.statistics.management.commands.process_purchase_files import Command
```

Comment:
This imports a class of name `Command`, and a class of the same name is declared in this file. This could lead to errors and is confusing: it is unclear what `Command.parse_german_number` refers to.
Instead you can do `import Command as ProcessPurchaseFilesCommand` to have a distinctive name for the command you import from.